db/data_migration/20170818104227_convert_unknown_news_articles_to_valid_news_articles.rb

```diff
@@ -0,0 +1,41 @@
+begin
+  # Update NewsArticles in Whitehall
+  unknown_news_articles = NewsArticle.by_subtype(NewsArticleType::Unknown)
+  document_ids = unknown_news_articles.map { |n| n.document_id }.uniq
+  press_releases = unknown_news_articles.includes(:document).select { |n| n.slug.include?("press-release") }
+  news_stories = unknown_news_articles - press_releases
+
+  press_releases.each do |news_article|
```

Comment:
Could we do a couple of `update_all`s here instead. I think that would just issue one SQL statement and avoid the validation that you're skipping on 18. Maybe `press_releases.update_all(news_article_type_id: NewsArticleType::PressRelease.id)` and then `press_releases.where(minor_change: false, change_note: nil).update_all(change_note: <etc>)`? No problem if you don't want to make this change but it would be much faster if it is taking a long time to run.